src/game/features/vehicle/FastTrain.cpp

```diff
@@ -0,0 +1,46 @@
+#include "core/commands/LoopedCommand.hpp"
+#include "core/frontend/Notifications.hpp"
+#include "game/features/Features.hpp"
+#include "game/rdr/Enums.hpp"
+#include "game/rdr/Natives.hpp"
+#include "game/backend/Self.hpp"
+
+namespace YimMenu::Features
+{
+	class FastTrain : public LoopedCommand
+	{
+		using LoopedCommand::LoopedCommand;
+		bool isDecelerating = false;
```

Comment:
If this isn't being used anymore, it can be removed.

Reply:
Good catch, I was trying to make it so when the OnDisable is triggered, it would decelerate from max speed to 0 gradually, it did not seem to work as intended though, if anyone has any idea how to achieve this, feel free.